contrib/exemplar_util/src/main/java/io/opencensus/contrib/exemplar/util/ExemplarUtils.java

```diff
@@ -50,31 +51,20 @@
   public static final String ATTACHMENT_KEY_SPAN_ID = "SpanId";
 
   /**
-   * Encodes a {@link TraceId} to a string in base 16 lower case encoding, and puts the encoded
-   * string into the attachments of the given {@link MeasureMap}.
+   * Puts a {@link SpanContext} into the attachments of the given {@link MeasureMap}.
    *
-   * @param measureMap the {@code MeasureMap}
-   * @param traceId the {@code TraceId} to be encoded and put.
-   * @since 0.16
-   */
-  public static void putTraceIdAsAttachment(MeasureMap measureMap, TraceId traceId) {
-    checkNotNull(measureMap, "measureMap");
-    checkNotNull(traceId, "traceId");
-    measureMap.putAttachment(ATTACHMENT_KEY_TRACE_ID, traceId.toLowerBase16());
-  }
-
-  /**
-   * Encodes a {@link SpanId} to a string in base 16 lower case encoding, and puts the encoded
-   * string into the attachments of the given {@link MeasureMap}.
+   * <p>{@link TraceId} and {@link SpanId} of the {@link SpanContext} will be encoded in base 16
+   * lower case encoding.
    *
    * @param measureMap the {@code MeasureMap}
-   * @param spanId the {@code SpanId} to be encoded and put.
+   * @param spanContext the {@code SpanContext} to be put as attachments.
    * @since 0.16
    */
-  public static void putSpanIdAsAttachment(MeasureMap measureMap, SpanId spanId) {
+  public static void putSpanContextAsAttachment(MeasureMap measureMap, SpanContext spanContext) {
```

Comment:
Suggestion: I think `putSpanContextAsAttachments` (or `putSpanContextAttachments`) would be more accurate, since this method adds multiple fields.

Reply:
SG, done.